test/Shared/StrongNameTests.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma warning disable IDE0005
```

Comment:
please add the comment here that describes what is being disabled.

Reply:
Will do; TL;DR - in some projects `Xunit` seemed to be already available as an implicit using, which caused the warning.